lib/spoom/cli/symbol_printer.rb

```diff
@@ -1,71 +0,0 @@
-# typed: true
-# frozen_string_literal: true
-
-require "colorize"
-
-module Spoom
-  module Cli
-    class SymbolPrinter
-      attr_accessor :seen, :no_color
-
-      def self.print_list(list, no_color)
-        printer = SymbolPrinter.new(2, no_color)
-        list.each do |item|
-          printer.print(" * ")
-          printer.visit(item)
-          printer.printn
-        end
-      end
-
-      def self.print_object(object, no_color)
-        printer = SymbolPrinter.new(2, no_color)
-        printer.visit(object)
-      end
-
-      def initialize(default_indent, no_color = false)
-        @seen = Set.new
-        @current_indent = default_indent
-        @no_color = no_color
-        String.disable_colorization = no_color
```

Comment:
Removing this is the big win: it disabled coloring at the app level.

This was a hack when I moved the LSP prototype to `spoom`.

We don't need it anymore.